astro/src/content/docs/get-started/run-in-the-cloud/cloud.mdx

```diff
@@ -810,3 +811,4 @@ Since it is a managed service, there are additional limitations as well:
 * You may not modify the Elasticsearch settings or view the Elasticsearch index directly. Among other things, this means that you can't use some of the [troubleshooting steps](/docs/operate/troubleshooting/troubleshooting) available to users self-hosting FusionAuth.
 * OpenTelemetry data is not available on FusionAuth Cloud deployments.
 * There is a limit of 1000 indexed fields. These include `user.data`, `registration.data` and standard indexed fields like `email`.
+* In certain cases, only current log files are available for download under <Breadcrumb>System -> Logs</Breadcrumb>. If you need all log files, including those previously rotated, please [open a support ticket](https://account.fusionauth.io/account/support/).
```

Comment:
It's pretty hard to answer that question, which is why I'm ok with the ambiguity.

You know you need to call support if you scroll thru the logs and it doesn't go back as far as you want.

Reply:
Yeah, we don't want to pull back the curtain too much because our customers shouldn't really care where their deployment lives. Ran into the same issue here: https://github.com/FusionAuth/fusionauth-site/pull/3235